app/editor/src/components/formik/form/FormikForm.tsx

```diff
@@ -7,15 +9,40 @@ export type FormikFormProps<
   ExtraProps = {},
 > = FormikConfig<Values> & ExtraProps;
 
-export interface IFormikFormProps<Values> extends FormikFormProps<Values> {}
+export interface IFormikFormProps<Values> extends FormikFormProps<Values> {
+  /**
+   * Manually control when the loading spinner overlay is visible.
+   */
+  loading?: boolean;
+  /**
+   * Configuration options for loading spinner overlay.
+   */
+  load?: {
+    hasRequests: boolean;
+    size?: string;
+    variant?: SpinnerVariant;
+  };
+}
 
 export const FormikForm = <Values extends FormikValues = FormikValues>({
```

Comment:
By default it now will show a loading overlay if there are active AJAX requests.